pop-api/src/v0/assets/mod.rs

```diff
@@ -0,0 +1,275 @@
+
+pub mod fungibles;
+
+const ASSETS_MODULE: u8 = 52;
+const VERSION: u8 = 0;
+
+/// [Pallet Assets](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/assets/src/lib.rs):
+/// 1. Dispatchables
+/// 2. Read state functions
+///
+/// 1. Dispatchables within pallet assets (TrustBackedAssets instance):
+const DISPATCH: u8 = 0;
+/// - create
+/// - start_destroy
+/// - destroy_accounts
+/// - destroy_approvals
+/// - finish_destroy
+/// - mint
+/// - burn
+/// - transfer
+/// - transfer_keep_alive
+const TRANSFER_KEEP_ALIVE: u8 = 9;
+/// - set_metadata
+/// - clear_metadata
+/// - approve_transfer
+const APPROVE_TRANSFER: u8 = 22;
+/// - cancel_approval
+const CANCEL_APPROVAL: u8 = 23;
+/// - transfer_approved
+const TRANSFER_APPROVED: u8 = 25;
+
+/// Issue a new class of fungible assets from a public origin.
+// pub(crate) fn create(
+// 	id: AssetId,
+// 	admin: impl Into<MultiAddress<AccountId, ()>>,
+// 	min_balance: Balance,
+// ) -> Result<()> {
+// 	dispatch(RuntimeCall::Assets(AssetsCall::Create {
+// 		id: id.into(),
+// 		admin: admin.into(),
+// 		min_balance,
+// 	}))
+// }
+//
+// /// Start the process of destroying a fungible asset class.
+// pub(crate) fn start_destroy(id: AssetId) -> Result<()> {
+// 	dispatch(RuntimeCall::Assets(AssetsCall::StartDestroy { id: id.into() }))
+// }
+//
+// /// Destroy all accounts associated with a given asset.
+// pub(crate) fn destroy_accounts(id: AssetId) -> Result<()> {
+// 	dispatch(RuntimeCall::Assets(AssetsCall::DestroyAccounts { id: id.into() }))
+// }
+//
+// /// Destroy all approvals associated with a given asset up to the max (see runtime configuration Assets `RemoveItemsLimit`).
+// pub(crate) fn destroy_approvals(id: AssetId) -> Result<()> {
+// 	dispatch(RuntimeCall::Assets(AssetsCall::DestroyApprovals { id: id.into() }))
+// }
+//
+// /// Complete destroying asset and unreserve currency.
+// pub(crate) fn finish_destroy(id: AssetId) -> Result<()> {
+// 	dispatch(RuntimeCall::Assets(AssetsCall::FinishDestroy { id: id.into() }))
+// }
+
+// /// Mint assets of a particular class.
+// pub(crate) fn mint(
+// 	id: AssetId,
+// 	beneficiary: impl Into<MultiAddress<AccountId, ()>>,
+// 	amount: Balance,
+// ) -> Result<()> {
+// 	dispatch(RuntimeCall::Assets(AssetsCall::Mint {
+// 		id: id.into(),
+// 		beneficiary: beneficiary.into(),
+// 		amount: Compact(amount),
+// 	}))
+// }
+//
+// /// Reduce the balance of `who` by as much as possible up to `amount` assets of `id`.
+// pub(crate) fn burn(id: AssetId, who: impl Into<MultiAddress<AccountId, ()>>, amount: Balance) -> Result<()> {
+// 	dispatch(RuntimeCall::Assets(AssetsCall::Burn {
+// 		id: id.into(),
+// 		who: who.into(),
+// 		amount: Compact(amount),
+// 	}))
+// }
+
+/// Move some assets from the sender account to another, keeping the sender account alive.
+#[inline]
+pub fn transfer_keep_alive(id: AssetId, target: AccountId, amount: Balance) -> Result<()> {
+	ChainExtensionMethod::build(u32::from_le_bytes([
```

Comment:
Added to #94, great suggestion

Reply:
Taking it a bit further, it could probably be 

```
fn build(version: u8, function: u8, module: u8, dispatchable: u8) -> ChainExtensionMethod {
   ChainExtensionMethod::build(u32::from_le_bytes([version, function, module, dispatchable]))
}
```

Reply:
@evilrobot-01 @Daanvdplas PR for refracting is here: https://github.com/r0gue-io/pop-node/pull/121
Please take a look and let me know if there are any feedback